app/services/model_service.py

```diff
@@ -26,14 +27,19 @@
 class ModelService:
     @staticmethod
     def get_model(model_id: str) -> Optional[ModelMetadata]:
+        try:
+            f = open(model_id + ".txt", "r")
```

Comment:
It seems that the opened file needs to be closed after being read.
FYI, you can open it with a `with` clause (https://www.pythonforbeginners.com/files/with-statement-in-python).